src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEMethodSymbol.vb

```diff
@@ -178,6 +192,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
                 Debug.Assert(BitsAreUnsetOrSame(_bits, bitsToSet))
                 ThreadSafeFlagOperations.Set(_bits, bitsToSet)
             End Sub
+
+            Public Sub SetIsUnmanagedCallersOnly(isUnmanagedCallersOnly As Boolean)
```

Comment:
>SetIsUnmanagedCallersOnly [](start = 23, length = 25)

"InitializeIsUnmanagedCallersOnly"? Consider following naming convention for other methods changing a pair of related bits.

Reply:
No longer initializing a pair of related bits.

---
In reply to: [566915186](https://github.com/dotnet/roslyn/pull/50886#discussion_r566915186) [](ancestors = 566915186)